data/json/items/tool/fire.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "id": "flint_steel",
+    "type": "TOOL",
+    "name": "flint and steel",
```

Comment:
This needs a way to require tinder for firestarting.

Reply:
I'm thinking add a flag for all kinds of inefficient firestarters that makes them require tinder. How do we want to define tinder here? By volume of flammable items with some stuff like gasoline or gunpowder specified too?

Reply:
What I've wanted to do in this space for a long time now is make a craftable prepared fire and use that to track how hard it is to start it.
Your basic pile of wood would require a sustained flame to catch it (though it might have other good properties).
A carefully stacked campfire with kindling might mitigate some wind penalties as well as allow spark-based ignition.
Other options are big whole-log fires that can burn without maintenance for hours.
This would also move the fuel consumption code out of fields.cpp and make it easier to tune for the "prepared fire" use case.

A simpler option would just add a "REQUIRES_TINDER" flag though :man_shrugging:

Reply:
Implemented TINDER and REQUIRES_TINDER flags. Tinder and gunpowder can be used right now.